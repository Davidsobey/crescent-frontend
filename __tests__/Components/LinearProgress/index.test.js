import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import configureStore from 'redux-mock-store'; // Smart components

// Component to be tested
import LinearProgress from '../../../src/Components/LinearProgress/index.js';


const mockStore = configureStore();
const initialState = {};
const store = mockStore(initialState);

describe('<LinearProgress />', () => {
  describe('render()', () => {
    test('renders the component', () => {
      const wrapper = shallow(<LinearProgress store={store} color="#000" />);
      const component = wrapper.dive();

      expect(toJson(component)).toMatchSnapshot();
    });
  });
});