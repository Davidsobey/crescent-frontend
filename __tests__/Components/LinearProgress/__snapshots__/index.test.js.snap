// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LinearProgress /> render() renders the component 1`] = `
<div
  className="LinearDeterminate-root-1"
>
  <WithStyles(LinearProgress)
    color="#000"
    value={0}
  />
</div>
`;
